src/utils.py

```diff
@@ -86,7 +86,7 @@ def process_tpf_query_text(resp, raw_repo_uri, call_name, extraMetadata):
         pagination_param['type'] = "int"
         pagination_param['in'] = "query"
         pagination_param['description'] = "The page number for this paginated query ({} results per page)".format(pagination)
-        params.append(pagination_param)
+        # params.append(pagination_param) # Does this get used elsewhere?
```

Comment:
But I think `params` is not declared anywhere in this method (I just noticed because my editor highlighted it as a warning), so not sure it is an issue